canonical-web-and-design/ubuntu.com
# Integrate smartquotes package

## Done
- Integrate smartquotes package
- Change one quote on the homepage for testing

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Find on the page "Ubuntu supports the fastest" and the That’s at the beginning on the sentence should look nice even though its a straight quote in the markup.
- Insure there is no errors or page slowness
